Packs/ThreatQ/Integrations/integration-ThreatQ.yml

```diff
@@ -2,10 +2,10 @@ commonfields:
   id: ThreatQ_Beta
   version: -1
 name: ThreatQ_Beta
-display: ThreatQ_Beta
+display: ThreatQ_Beta (Deprecated)
```

Comment:
WDYT?
```suggestion
display: ThreatQ (Deprecated)
```

Reply:
Yea you're right. No need for the beta in the display anymore.